x/programs/cmd/simulator/cmd/key.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ava-labs/avalanchego/database"
+
+	"github.com/ava-labs/hypersdk/crypto/ed25519"
+	"github.com/ava-labs/hypersdk/state"
+	"github.com/ava-labs/hypersdk/utils"
+)
+
+const (
+	// keyPrefix that stores pub key -> private key mapping
+	keyPrefix = 0x1
```

Comment:
If there is more than 1 prefix, I would recommend defining all of them in a central location.